.github/workflows/stale.yml

```diff
@@ -1,8 +1,14 @@
 name: 'Close stale issues and PRs'
 on:
   schedule:
-    - cron: '30 1 * * *'
+    - cron: '30 * * * *'
```

Comment:
This seems too frequent. The original daily cleanup is good enough for us. The previous issue is more about the limited processing number of issues instead of frequency.

Reply:
Currently, the operation is not enough to go through all issues, but increasing operation might hit rate limit. In order to prevent rate limit, we can increase the frequency to check all issues.

After we check all issues and close most unactivated issues, we could tune the frequency back.